Makefile

```diff
@@ -1,95 +1,105 @@
-.PHONY: minikube-start minikube-stop minikube-delete minikube-dashboard minikube-tunnel init-local init-staging init-production
-
+.PHONY: help
+help: # @HELP Print this message
+help:
+	@echo "TARGETS:"
+	@grep -E '^.*: *# *@HELP' $(MAKEFILE_LIST)    \
+	    | awk '                                   \
+	        BEGIN {FS = ": *# *@HELP"};           \
+	        { printf "  %-20s %s\n", $$1, $$2 };  \
+	    '
+
+.PHONY: minikube-start
+minikube-start: # @HELP Start a local k8s cluster using minikube
 minikube-start:
-	# version 1.21.4 is currently used in the production environments
+# version 1.21.4 is currently used in the production environments
```

Comment:
I think I did change it so that it becomes a Makefile level comment (no indent) instead of a bash one (with indent), but as it's outdated anyways, let's get rid of it.